bot/constants.py

```diff
@@ -16,7 +17,7 @@ class AdventOfCode:
     leaderboard_id = 363275
     leaderboard_join_code = str(environ.get("AOC_JOIN_CODE", None))
     leaderboard_max_displayed_members = 10
-    year = 2018
+    year = datetime.utcnow().year
```

Comment:
More for SA than anything else since it doesn't functionally affect the functionality of the PR: the TZ of AOC mirrors that of its maintainer (EST, UTC-5).

Reply:
this is because seasonal bot uses this TZ. @SebastiaanZ do you have a comment?

Reply:
I don't really have an opinion on this. 

This value is equal the whole year round and AoC does not overlap with a change of its value, so I don't think it's really that relevant for us if our year change syncs perfectly with the year change of the AoC servers.